aopy/visualization/base.py

```diff
@@ -2498,4 +2498,115 @@ def plot_plane(plane, ax=None, gain=1.0, color='grey', alpha=0.15, resolution=10
         raise ValueError(f"Invalid mapping shape {plane.shape}. Expected (3,3) or (4,4) \
                          for transformation matrices, or (4,) for plane equations.")
         
-    ax.plot_surface(x, y, z, alpha=alpha, color=color)
\ No newline at end of file
+    ax.plot_surface(x, y, z, alpha=alpha, color=color)
+
+def plot_sphere(location, color='gray', radius=4, resolution=20, alpha=1, ax=None, **kwargs):
```

Comment:
this **kwargs isn't used

Reply:
it would be nice to include a `bounds` argument to both these functions. see plot_targets for example